_sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md

```diff
@@ -0,0 +1,8 @@
+{:.do}
+### Wear a facemask when in public
+
+Studies have shown for Influenza that a mask may have a self-protection benefit. The viron that causes COVID-19 is roughly the same size as the Influenza viron at 125 nanometers and ithas the same mechanisms of entering the human body. If you absolutely have to venture into public where other people may be, buy a cloth mask or make your own as US hospital chains are beginning to do. 
```

Comment:
"Viron" is scientific jargon that many readers won't understand. Maybe we could use the word "virus" even though it isn't as correct. Alternatively, we could add a parenthetical, like so:

"The viron (complete virus particle) that causes COVID-19 is roughly the same size..."

Reply:
+1

(I like either suggestion)